### ISSUE TITLE
Docs: word is covered up ? 

### ISSUE THREAD

**Issue by shezhangzhang** [getsentry/develop#677]

**Comment by chadwhitacre** [comment]
Hrm, something funky going on. Looks fine [in GitHub's rendering](https://github.com/getsentry/develop/blob/master/src/docs/architecture.mdx).

🤔

**Comment by shezhangzhang** [comment]
Yeah, I found it too...🤪 GitHub looks fine. Sentry docs looks wrong.

**Comment by chadwhitacre** [comment]
It's an SVG, is there some CSS that is wrong?

**Comment by shezhangzhang** [comment]
@chadwhitacre I have no idea, maybe something is wrong with this repo: https://github.com/remcohaszing/gatsby-remark-mermaid
